ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -186,10 +189,15 @@ export default defineComponent({
       const { data } = await fetchMhRegistration(getMhrInformation.value.mhrNumber)
       const currentOwnerGroups = data?.ownerGroups || [] // Safety check. Should always have ownerGroups
       // Create an ID to each individual owner for UI Tracking
+      // TODO: Remove after API updates to include the ID for Owners
       currentOwnerGroups.forEach(ownerGroup => {
+        ownerGroup.isCurrentOwnerGroup = true
```

Comment:
Same for this property, not sure it is required or used here. Perhaps left over from some testing?

Reply:
Also not needed. Removed.